README.md

```diff
@@ -180,6 +180,125 @@ git checkout -b 0.3_kovan_deployment
 
 
+# Airnode Setup Guide
+
+## Setup
+
+install the dependencies before running the scripts
+
+```
+yarn install
+```
+
+compile the contracts using the following command:
+
+```
+yarn build
+```
+
+## Instructions
+
+The following instructions will guide you step by step on how to deploy an airnode.
+
+The deployment receipts will be saved in the `/airnode-deployment` folder.
+
+### 1. Setup the deployment 
+
+The first step is to choose a network for deployment and provide the provider URL. You will also need a funded account to deploy the testing contracts that will be derived from the mnemonic in this step.
+
+Run the following script to setup the integration
+
+```
+yarn setup-integration
+```
+
+### 2. (Optional) Print out the user account information
+
+Run:
+```
+yarn print-account-details
+```
+This script will show you the address of the account derived from the specified mnemonic. This account will be used to deploy the test contract. Make sure it is funded. The recommended amount is at least 0.3 ETH ( or the gas token of the chain your deploying to eg MATIC for polygon mainnet) .
+
+### 3. Create AWS secrets file
+If you intend to deploy Airnode on AWS, you will need to specify the credentials which will be used by the deployer. If you are not sure where to find these or how to create an AWS account, [see the following docs section](https://docs.api3.org/airnode/v0.3/grp-providers/docker/deployer-image.html#aws).
+
+After you know the secrets, run the following script to specify them:
+```
+yarn create-aws-secrets
+```
+### 4. Create Airnode configuration
+
+Airnode is configured by two files - `config.json` and `secrets.env`. The configuration is different based on where the Airnode is deployed, because every cloud provider has different settings. These differences are minor and we take care of it for you.
+
+To generate the config.json, run:
+```
+yarn create-airnode-config
+```
+
+Note: This config.json is made for coingecko API to get the price of a token, You need to modify this file to use a different API.
+
+### 5. Create Airnode secrets
+
+Airnode is configured by two files - `config.json` and `secrets.env`. The config.json was already created in previous step. The latter, secrets.env can be generated it using:
+
+```
+yarn create-airnode-secrets
+```
+
+### 6. Build docker artifacts and deployer.
+
+Clone the [Airnode repository](https://github.com/api3dao/airnode) in a different folder.
+
+In the root of the Airnode repo run the following commands to install the dependencies and build the packages
+
+```
+yarn run bootstrap
+
+yarn run build
+```
+
+Then to build the artifacts run the following command, this command can take a while to complete
+
+```
+yarn docker:build:artifacts
+```
+
+To build the deployer run the following command
+
+```
+yarn docker:build:deployer
+```
+
+Once both of these images have been built head back to the beacon-setup-guide repository.
```

Comment:
Thankfully this can all be avoided as the subsequent step `deploy-airnode` uses the existing `0.3.1` image.
```suggestion
```